.github/dependabot.yml

```diff
@@ -6,3 +6,6 @@ updates:
       interval: daily
     open-pull-requests-limit: 100
     registries: "*"
+    labels:
+      # We do not need changelogs because all dependencies are currently dev dependencies
+      - 🤖Skip Changelog'
```

Comment:
Is this correct with the robot emoji and the single quote at the end?

Reply:
Robot Emoji is correct. Quote at only the end is my bad.

Reply:
Fixed!  [Here's a link](https://github.com/Shopify/i18next-shopify/blob/main/.github/workflows/changelog.yml#L16) to where the label is referenced.